modules/t/MultiTestDB.t

```diff
@@ -74,4 +74,9 @@ ok(scalar(@{$sth->fetchall_arrayref}) == 21);
 
 $sth->finish;
 
+# check Parus major test database connection
```

Comment:
Not sure if this test is required here.

Reply:
I would like to add in checks that this Parus major test database can be loaded correctly (particularly via the Travis-CI build) now that I've updated the schema version field because there were loading errors from Travis-CI prior to this change. Is there a better place for me to add in such a test?